### PR TITLE
Removed labeled-by attribute from file input

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
#222

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix)

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

File: src/app/public/modules/file-attachment/file-attachment.component.spec.ts [hunk 1]
```diff
@@ -1083,6 +1083,14 @@ describe('File attachment', () => {
       expect(fixture.nativeElement).toBeAccessible();
     });
   }));
+
+  it('should pass accessibility when label changes', async(() => {
+    fixture.componentInstance.labelText = 'Something different';
+    fixture.detectChanges();
+    fixture.whenStable().then(() => {
+      expect(fixture.nativeElement).toBeAccessible();
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/file-attachment/file-attachment.component.spec.ts:1091]: We should now be using `expectAsync` on these.

File: src/app/public/modules/file-attachment/file-attachment.component.spec.ts [hunk 2]
```diff
@@ -1083,6 +1083,14 @@ describe('File attachment', () => {
       expect(fixture.nativeElement).toBeAccessible();
     });
   }));
+
+  it('should pass accessibility when label changes', async(() => {
+    fixture.componentInstance.labelText = 'Something different';
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/file-attachment/file-attachment.component.spec.ts:1088]: We say `when label changes` but are only testing the "after". I know we are testing the "before" in the test before but would you mind calling that out in a quick code comment there?

File: src/app/public/modules/file-attachment/file-attachment.component.spec.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/file-attachment/file-attachment.component.spec.ts:1091]: See https://github.com/blackbaud/skyux-indicators/blob/65ba36456b769183a1102842438302340307d9e2/src/app/public/modules/wait/wait.component.spec.ts#L50

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
@Blackbaud-AlexKingman - just a couple small comments.

File: src/app/public/modules/file-attachment/fixtures/file-attachment.component.fixture.ts [hunk 3]
```diff
@@ -24,6 +24,8 @@ export class FileAttachmentTestComponent implements OnInit {
 
   public fileForm: FormGroup;
 
+  public labelText: string = 'Choose File';
```
Comment (blackbaud-johnly) [inline src/app/public/modules/file-attachment/fixtures/file-attachment.component.fixture.ts:27]:
```suggestion
  public labelText: string = 'Choose file';
```

File: src/app/public/modules/file-attachment/file-attachment.component.spec.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/file-attachment/file-attachment.component.spec.ts]: Done.

File: src/app/public/modules/file-attachment/file-attachment.component.spec.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/file-attachment/file-attachment.component.spec.ts:1089]: Updated the comments to be more clear.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-file-label-fix)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]
LGTM

**Pull request merged**